precompiles/common/precompiles.go

```diff
@@ -74,3 +74,12 @@ func HandlePayment(ctx sdk.Context, precompileAddr sdk.AccAddress, payer sdk.Acc
 	}
 	return coin, nil
 }
+
+func HandlePaymentUseiWei(ctx sdk.Context, precompileAddr sdk.AccAddress, payer sdk.AccAddress, value *big.Int, bankKeeper BankKeeper) (sdk.Int, sdk.Int, error) {
+	usei, wei := state.SplitUseiWeiAmount(value)
+	// refund payer because the following precompile logic will debit the payments from payer's account
+	if err := bankKeeper.SendCoinsAndWei(ctx, precompileAddr, payer, usei, wei); err != nil {
+		return sdk.Int{}, sdk.Int{}, err
```

Comment:
NOTE: We can also just not use this `HandlePaymentUseiWei` for `sendNative` and directly bank send from the precompileAddr to the receiver addr. Just left it in case we specifically for it to be debited from user's sei address

Reply:
directly sending should be fine, but we can leave it as is in this PR for now